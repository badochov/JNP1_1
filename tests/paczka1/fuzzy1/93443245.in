?
S754 S744 96,4
PalvUIssl0j S754 40,6
EObe A684 64,2
PalvUIssl0j S754 15,8
EObe A684 184,9
gMI S754 85,3
S754 A684 86,9
n58ei S85 63,7
S754 A684 392,8

S754 S85 91,6
n58ei S85 291,7
n58ei S85 89,9
S754 S85 306,9
EObe A684 95,7
?A684
n58ei S85 18,1
wfEqG3NQn S754 85,5
n58ei S85 57,9
PalvUIssl0j S754 19,8
wfEqG3NQn S85 39,3
PalvUIssl0j S744 10,0
EObe S744 78,5
n58ei A684 72,8
wfEqG3NQn S85 50,7
gMI S754 342,9
zEshg7uPE A684 14,5
EObe S744 56,7
zEshg7uPE A684 59,5

EObe S754 37,5
PalvUIssl0j S754 61,0
?S744
PalvUIssl0j S754 154,9
n58ei S85 67,0
gMI S754 18,1
?gMI
zEshg7uPE A684 291,0
n58ei S85 333,0