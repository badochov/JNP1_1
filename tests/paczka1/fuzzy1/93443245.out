A684 426,6
S744 21,8
gMI S 257,6
